.github/workflows/build.yml

```diff
@@ -7,37 +7,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: jitterbit/get-changed-files@v1
-        id: files
         with:
-          format: space-delimited
-          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
What happens to the push event?

Reply:
I updated PR with using specific SHA, It works on both push and pull_request events now